lib/Interpreter/CIFactory.cpp

```diff
@@ -626,7 +631,15 @@ namespace {
       argvCompile.insert(argvCompile.begin() + 2, lang);
     }
 
-    AddHostCXXIncludes(argvCompile);
+    std::string clang = llvm::sys::fs::getMainExecutable("cling", (void*)(intptr_t) locate_cling_executable);
+    {
```

Comment:
Remove the nested scope. Please also add a comment explaining that you are looking freshly built clang sitting next to cling.